### PR TITLE
Initial delay splunk events

### PR DESCRIPTION
**Pull request by craffit**
Add possibility of adding initial delay before starting to query splunk events

### CODE REVIEW

File: checks.d/splunk_event.py [hunk 1]
```diff
@@ -78,7 +78,10 @@ def __init__(self, instance, init_config):
         self.tags = instance.get('tags', [])
         self.initial_delay_first_time_seconds = int(instance.get('initial_delay_first_time_seconds', self.instance_config.default_initial_delay_first_time_seconds))
 
+        self.launch_time_seconds = current_time
 
+    def initial_time_done(self, current_time):
+        return current_time >= self.launch_time_seconds + self.instance_config.default_initial_delay_first_time_seconds
```
Comment (fmeeuw) [inline checks.d/splunk_event.py]: Shouldn;t it be self.initial_delay_first_time_seconds (Without the default)

File: checks.d/splunk_event.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def __init__(self, instance, init_config):
 class Instance:
     INSTANCE_TYPE = "splunk"
 
-    def __init__(self, instance, init_config):
+    def __init__(self, current_time, instance, init_config):
```
Comment (fmeeuw) [inline checks.d/splunk_event.py]: I would prefer to name it current_time_seconds

File: tests/checks/mock/test_splunk_event.py [hunk 3]
```diff
@@ -275,6 +275,60 @@ def json(self):
         self.assertEquals(self.service_checks[0]['status'], 2, "service check should have status AgentCheck.CRITICAL")
 
 
+class TestSplunkDelayFirstTime(AgentCheckTest):
```
Comment (fmeeuw) [inline tests/checks/mock/test_splunk_event.py:284]: Good test :)

**Review by fmeeuw — CHANGES_REQUESTED** [review]

File: checks.d/splunk_event.py (hunk 2 above)
Comment (craffit) [reply checks.d/splunk_event.py]: done

File: checks.d/splunk_event.py (hunk 1 above)
Comment (craffit) [reply checks.d/splunk_event.py]: good catch

**Pull request merged**